### PR TITLE
Pure action

### LINKED ISSUE #103: Remove backing web app
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the feature has not already been requested

### 🚀 Feature Proposal

GitHub announced that dependabot will now respect the GITHUB_TOKEN permissions configured in workflows.

https://github.blog/changelog/2021-10-06-github-actions-workflows-triggered-by-dependabot-prs-will-respect-permissions-key-in-workflows/

Because of this, there's a good chance that this action won't need the backing web app it's currently using and can be rewritten as a plain, self-contained action.

### Motivation

_No response_

### Example

_No response_

### PR DESCRIPTION
**Pull request by Eomm**
fixes #103 

Since this PR will be a major release I would propose to set the `target` default value from `major` to `minor`.

### CODE REVIEW

**Comment by simoneb** [comment]
> fixes #103
> 
> Since this PR will be a major release I would propose to set the `target` default value from `major` to `minor`.

I would not change the semantics of this action.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -7,14 +7,17 @@ jobs:
       - uses: actions/checkout@v2.4.0
       - uses: actions/setup-node@v2.4.1
         with:
-          node-version: '14.x'
+          node-version: '12.x' # https://github.com/actions/runner/issues/772
```
Comment (simoneb) [inline .github/workflows/ci.yml:10]:
```suggestion
          node-version: 12 # https://github.com/actions/runner/issues/772
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -7,14 +7,17 @@ jobs:
...
       - run: npm ci
       - run: npm run lint
       - run: npm run test
 
   automerge:
     needs: build
     runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+      contents: write
```
Comment (simoneb) [inline .github/workflows/ci.yml:20]: why is this needed?

File: .nvmrc [hunk 3]
```diff
@@ -0,0 +1 @@
+12
```
Comment (simoneb) [inline .nvmrc:1]: new line

File: README.md [hunk 4]
```diff
@@ -29,10 +25,6 @@ _Optional_ The merge method you would like to use (squash, merge, rebase). Defau
 
 _Optional_ An arbitrary message that you'd like to comment on the PR after it gets auto-merged. This is only useful when you're recieving too much of noise in email and would like to filter mails for PRs that got automatically merged.
 
-### `api-url`
-
-_Optional_ A custom url where the external API which is delegated the task of approving and merging responds.
-
 ### `target`
 
 _Optional_ A flag to only auto-merge updates based on Semantic Versioning. Default to `major` merge. Possible options are:
```
Comment (simoneb) [inline README.md:30]: there is an inconsistency here and in the next line. default is now any, not major

File: package.json [hunk 5]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "build": "ncc build src/index.js",
     "lint": "eslint .",
-    "test": "tap",
+    "test": "tap test/*.test.js",
```
Comment (simoneb) [inline package.json:9]: why this change?

Comment (Eomm) [reply package.json:9]: locally without it, the command tooks ~2/3 seconds more to start

Comment (simoneb) [reply package.json:9]: did you figure out why?
